tests/fast-integration/eventing-test/utils.js

```diff
@@ -121,11 +130,26 @@ async function getStreamConfigForJetStream() {
   return envsCount === 2;
 }
 
+async function getJetStreamStreamData(host) {
+  const responseJson = await retryPromise(async () => await axios.get(`https://${host}/jsz?streams=true`), 5, 1000);
+  const streamName = responseJson.data.account_details[0].stream_detail[0].name;
+  const streamCreationTime = responseJson.data.account_details[0].stream_detail[0].created;
+
+  return {
+    streamName: streamName,
+    streamCreationTime: streamCreationTime,
+  };
+}
+
 function skipAtLeastOnceDeliveryTest() {
   return !(streamConfig['retention_policy'] === 'limits' &&
       streamConfig['consumer_deliver_policy'] === 'all');
 }
 
+function skipStreamReCreationTest(streamData) {
```

Comment:
This name to me implies that this function is skipping a test. Can you please rename this in a way that implies that it will return a bool? Something like `function isStreamRecreationTestSkippable(streamData)` or, because this is more what this function does, `isStreamCreationTimeMissing`. This will also create `if`s that are nicer to read.